cloudbuild.yaml

```diff
@@ -1,12 +1,16 @@
 steps:
   - id: 'build'
-    name: 'earthly/earthly:v0.5.24'
+    name: 'earthly/earthly:v0.7.23'
+    env:
+      - "HOME=/root"
     args:
```

Comment:
I would also add `--ci` and `--push` here since these are very common in CI. We use these in all our other CI examples.

Reply:
We can, I was just trying to do the minimum possible to make it go, ill see what this does

Reply:
Adding push would mean setting up and configuring a Dockerhub repository. Do we want that to be in the scope of this PR?

Reply:
That's fine, no `--push` then.